libs/modules/user-tasks/application/src/lib/user-tasks-service-factory.ts

```
@@ -0,0 +1,20 @@
+    if (!this.instance) {
+      this.instance = new UserTasksService(repository);
+    }
+  }
+
+  static getInstance(): UserTasksService {
+    if (!this.instance) {
+      throw new Error('UserTasksService must be initialized before use');
+    }
+    return this.instance;
+  }
+}
```

Biome (1.9.4)

[error] Avoid classes that contain only static members.

Prefer using simple functions instead of classes with only static members.

(lint/complexity/noStaticOnlyClass)